exercises/sam_jenkins/ex01_basics/src/stringManip.cpp

```diff
@@ -0,0 +1,25 @@
+#include <string>
+#include <cctype>
+#include <regex>
+using std::string;
+
+
+bool space(const char& c)
+{
+	return (isspace(c) || c == '-');
+}
+
+
+void strip_string(string& input)
```

Comment:
This name is find, but in the project naming conventions are camel case: `stripString`